examples/fc_networks.py

```diff
@@ -53,64 +51,75 @@
 
 oneview_client = OneViewClient(config)
 
+connection = oneview_client.connection
+
+fc_network = FcNetworks(connection, options)
+
+additional_params = {}
+fc_network.create(additional_params)
+
+#Get the data of a fc network
+fc_netwok_data = fc_network.get()
+
+
 # Create a FC Network
-fc_network = oneview_client.fc_networks.create(options)
+#fc_network = oneview_client.fc_networks.create(options)
 print("\nCreated fc-network '%s' successfully.\n  uri = '%s'" % (fc_network['name'], fc_network['uri']))
 
 # Find recently created network by name
-fc_network = oneview_client.fc_networks.get_by('name', 'OneViewSDK Test FC Network')[0]
+#fc_network = oneview_client.fc_networks.get_by('name', 'OneViewSDK Test FC Network')[0]
 print("\nFound fc-network by name: '%s'.\n  uri = '%s'" % (fc_network['name'], fc_network['uri']))
 
 # Update autoLoginRedistribution from recently created network
-fc_network['autoLoginRedistribution'] = False
-fc_network = oneview_client.fc_networks.update(fc_network)
+#fc_network['autoLoginRedistribution'] = False
+#fc_network = oneview_client.fc_networks.update(fc_network)
```

Comment:
Why this update call is commented ?

Reply:
Looks like you are looking into older commits. Those lines are not commented in latest commit.